spring-cloud-gcp-examples/spring-cloud-gcp-config-example/src/main/resources/bootstrap.properties

```diff
@@ -0,0 +1,16 @@
+
+# Sample bootstrap configuration
+
+# `spring.cloud.config.name` will be used if spring.cloud.gcp.config.name is not provided
+# spring.application.name is used if neither spring.cloud.config.name or spring.cloud.gcp.config.name is provided
+spring.cloud.gcp.config.name=myapp
+
+# `spring.cloud.config.profile` will be used if spring.cloud.gcp.config.profile is not provided
+spring.cloud.gcp.config.profile=prod
+
+# Config server API timeout in milliseconds, default : 18000 ( 3 minutes)
+spring.cloud.gcp.config.timeout = 12000
```

Comment:
Crazy nit: this equal separator has spaces around and the previous two don't. Sorry :)